pom.xml

```diff
@@ -3,7 +3,7 @@
     <parent>
     	<groupId>org.jenkins-ci.plugins</groupId>
     	<artifactId>plugin</artifactId>
-        <version>1.557</version>
+        <version>1.586</version>
```

Comment:
Is this really necessary? This will prevent use of the plugin in current LTS (1.580.x) releases.

See https://wiki.jenkins-ci.org/display/JENKINS/Choosing+Jenkins+version+to+build+against for more pros/cons.